Tests/ModelTest.php

```diff
@@ -33,18 +33,31 @@ public function testInsert()
 
 		$user = new User(self::$driver, $attributes);
 
-		$user->save();
+		$user->persist();
 
 		$this->assertEquals(
 			101,
 			$user->id
 		);
 
+		$attributes = [
+				'profile_key' => 'none',
+				'profile_value' => 0,
+				'ordering' => 0
+			];
+
+		$userProfile = new UserProfile(self::$driver, $attributes);
+
+		$user->profile()->save($userProfile);
```

Comment:
Shouldn't this be persist?

Reply:
That's for a relation, which saves a related Model, I suppose I can change it to persist for consistency.